🐛 Bug Report: Cannot find module '@novu/framework/next' or its corresponding type declarations.ts

### 📜 Description

Cannot find module '@novu/framework/next' or its corresponding type declarations.ts

### 👟 Reproduction steps
import { serve } from "@novu/framework/next";

"@novu/framework": "2.0.0-canary.5"

### 👍 Expected behavior

Typescript support

### 👎 Actual Behavior with Screenshots

Unable to import from '@novu/framework/next'

### Novu version

2.0.0-canary.5

### npm version

10.5.0

### node version

v18.20.1

### 📃 Provide any additional context for the Bug.

_No response_

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find a similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read the [Contributing Guidelines](https://github.com/novuhq/novu/blob/main/CONTRIBUTING.md)

### Are you willing to submit PR?

Yes I am willing to submit a PR!

--- a/packages/framework/package.json
+++ b/packages/framework/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@novu/framework",
-  "version": "2.3.0",
+  "version": "2.4.0",
   "description": "The Code-First Notifications Workflow SDK.",
   "main": "./dist/cjs/index.cjs",
   "types": "./dist/cjs/index.d.cts",

--- a/packages/nextjs/package.json
+++ b/packages/nextjs/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@novu/nextjs",
-  "version": "2.6.0",
+  "version": "2.6.1",
   "repository": "https://github.com/novuhq/novu",
   "description": "Novu's Next.js SDK for building custom inbox notification experiences",
   "author": "",

--- a/packages/shared/package.json
+++ b/packages/shared/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@novu/shared",
-  "version": "2.1.1",
+  "version": "2.1.2",
   "description": "",
   "scripts": {
     "start": "npm run start:dev",

